File: RhNetAPP/src/app/components/models/adm/user.model.ts
import {Profile} from 'src/app/components/models/adm/profile.model';
import { Client } from 'src/app/components/models/adm/client.model';
export interface User {
	username: string, 
	email: string,
	token: string,
	profiles: Profile[],
	currentClient: Client
}

File: RhNetAPI/RhNetAPI/Models/Adm/UserModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace RhNetAPI.Models.Adm
{
    public class UserModel
    {
        public string Username { get; set; }
        public string Email { get; set; }
        public string Token { get; set; }
        public List<RoleModel> Profiles { get; set; }
        public ClientModel CurrentClient { get; set; }
    }
}
